Octokit/Models/Response/RepositoryInvitation.cs

```diff
@@ -14,6 +14,10 @@ public enum InvitationPermissionType
     [DebuggerDisplay("{DebuggerDisplay,nq}")]
     public class RepositoryInvitation
     {
+        public RepositoryInvitation()
```

Comment:
for consistency with all the other response class ctors, can you put this on a single line?

`public RepositoryInvitation() { }`